includes/classes/Elasticsearch.php

```diff
@@ -1175,13 +1175,15 @@ public function remote_request( $path, $args = [], $query_args = [], $type = nul
 				 * Filter intercepted request
 				 *
 				 * @hook ep_do_intercept_request
+				 * @since 3.2.2
+				 * @since 4.0.0 added $type
 				 * @param {array} $request New remote request response
 				 * @param  {array} $query Remote request arguments
 				 * @param  {args} $args Request arguments
 				 * @param  {int} $failures Number of failures
```

Comment:
Missing annotation in the docblock

Reply:
Good catch! This is fixed now, @rinatkhaziev.